config/runtimes/vllm-multinode-template.yaml

```diff
@@ -177,33 +189,34 @@ objects:
         containers:
           - name: worker-container
             image: $(vllm-image)
-            command: ['bash', '-c']
-            args:
-              - |
-                SECONDS=0
+            command:
+            - "bash"
+            - "-c"
+            - |
+              SECONDS=0
 
-                while true; do              
-                  if (( SECONDS <= 240 )); then
-                    if ray health-check --address "${HEAD_SVC}.${POD_NAMESPACE}.svc.cluster.local:6379" > /dev/null 2>&1; then
-                      echo "Global Control Service(GCS) is ready."
-                      break
-                    fi
-                    echo "$SECONDS seconds elapsed: Waiting for Global Control Service(GCS) to be ready."
-                  else
-                    if ray health-check --address "${HEAD_SVC}.${POD_NAMESPACE}.svc.cluster.local:6379"; then
-                      echo "Global Control Service(GCS) is ready. Any error messages above can be safely ignored."
-                      break
-                    fi
-                    echo "$SECONDS seconds elapsed: Still waiting for Global Control Service(GCS) to be ready."
-                    echo "For troubleshooting, refer to the FAQ at https://docs.ray.io/en/master/cluster/kubernetes/troubleshooting/troubleshooting.html#kuberay-troubleshootin-guides"
+              while true; do
+                if (( SECONDS <= 240 )); then
+                  if ray health-check --address "${HEAD_SVC}.${POD_NAMESPACE}.svc.cluster.local:6379" > /dev/null 2>&1; then
+                    echo "Global Control Service(GCS) is ready."
+                    break
                   fi
+                  echo "$SECONDS seconds elapsed: Waiting for Global Control Service(GCS) to be ready."
+                else
+                  if ray health-check --address "${HEAD_SVC}.${POD_NAMESPACE}.svc.cluster.local:6379"; then
+                    echo "Global Control Service(GCS) is ready. Any error messages above can be safely ignored."
+                    break
+                  fi
+                  echo "$SECONDS seconds elapsed: Still waiting for Global Control Service(GCS) to be ready."
+                  echo "For troubleshooting, refer to the FAQ at https://docs.ray.io/en/master/cluster/kubernetes/troubleshooting/troubleshooting.html#kuberay-troubleshootin-guides"
+                fi
 
-                  sleep 5
-                done
+                sleep 5
+              done
 
-                export RAY_HEAD_ADDRESS="${HEAD_SVC}.${POD_NAMESPACE}.svc.cluster.local:6379"
-                echo "Attempting to connect to Ray cluster at $RAY_HEAD_ADDRESS ..."
-                ray start --address="${RAY_HEAD_ADDRESS}" --block
+              export RAY_HEAD_ADDRESS="${HEAD_SVC}.${POD_NAMESPACE}.svc.cluster.local:6379"
```

Comment:
isn't better to expose it in the beginning of this script and reuse in the if conditions?

Reply:
updated